internal/resource-request-operator/broadcaster.go

```diff
@@ -192,3 +300,23 @@ func updateResources(currentResources, oldResources, newResources corev1.Resourc
 		}
 	}
 }
+
+func setLabelSelectors(options *metav1.ListOptions) {
```

Comment:
As this function is also used in other parts of the code, I would generalize that and avoid creating a duplicate